include/ignition/rendering/ParticleEmitter.hh

```diff
@@ -208,6 +209,34 @@ namespace ignition
       public: virtual void SetColorRange(
                   const ignition::math::Color &_colorStart,
                   const ignition::math::Color &_colorEnd) = 0;
+
+      /// \brief Get the amount by which to scale the particles in both x and y
+      /// direction per second.
+      /// Default value is 1.
+      /// \return The scale rate.
+      /// \sa SetScaleRate
+      public: virtual double ScaleRate() const = 0;
+
+      /// \brief Set the amount by which to scale the particles in both x and y
+      /// direction per second.
+      /// \param[in] _scaleRate The scale rate.
+      /// \sa ScaleRate
+      public: virtual void SetScaleRate(double _scaleRate) = 0;
+
+      /// \brief Get the path to the color image used as an affector.
+      /// \return The color image name or empty string if the image is not set.
+      /// \sa SetColorImage
+      public: virtual std::string ColorImage() const = 0;
+
+      /// \brief Set the path to the color image used as an affector. This
+      /// affector modifies the colour of particles in flight. The colors are
+      /// taken from a specified image file. The range of colour values begins
+      /// from the left side of the image and move to the right over the
+      /// lifetime of the particle, therefore only the horizontal dimension of
+      /// the image is used.
+      /// \param[in] _image The color image name.
+      /// \sa ColorImage
+      public: virtual void SetColorImage(const std::string &_image) = 0;
```

Comment:
It makes sense. It would even make more sense to overload the functions but C++ doesn't allow us to overload two functions just based on the return value :/ 571a08f.